app/views/refinery/authentication/devise/admin/users/_user.html.erb

```diff
@@ -1,23 +1,27 @@
-<li id="sortable_<%= user.id %>" class='clearfix record <%= cycle("on", "on-hover") %>'>
-  <span class='title'>
-    <strong><%= user.username %></strong>
-    <span class="preview">
-      <%= t('.preview', :who => user.email, :created_at => l(user.created_at, :format => :short)) %>
+<li class='page-listing' id="<%= dom_id(user) -%>">
+  <div class="page-row">
+    <i class="icon icon-user-1"></i>&nbsp;
+
+    <% tag_name = current_refinery_user.can_edit?(user) ? :a : :div %>
+    <%= content_tag(tag_name, class: 'page-title', href: refinery.edit_authentication_devise_admin_user_path(user)) do %>
+      <span class='title'>
+        <%= user.username %>
+      </span>
+    <% end %>
+
+    <span class='actions'>
+      <% if current_refinery_user.can_delete?(user) %>
+        <span class="delete pull-right">
+          <%= link_to refinery.authentication_devise_admin_user_path(user),
+                      :class => "cancel confirm-delete circle-icon delete",
+                      :method => :delete,
+                      :data => {
+                        :confirm => t('message', :scope => 'refinery.authentication.devise.admin.delete', :title => user.username)
+                      } do %>
+            <i class="icon icon-trash bump-up-2"></i>
+          <% end %>
+        </span>
+      <% end %>
     </span>
-  </span>
-  <span class='actions'>
-    <%= mail_to user.email, refinery_icon_tag('email_go.png'),
```

Comment:
It's pretty simple.  We dropped the email button because finding an icon and testing that link (though simple) is just not worth the expected times this will be used.  That's our thought.